src/NuGet.Core/NuGet.ProjectModel/NuGet.ProjectModel.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);netstandard2.1</TargetFrameworks>
```

Comment:
Update the property in the common targets.

Reply:
Thanks! Changed accordingly.